### ISSUE TITLE
need documentation for new Response type

### ISSUE THREAD

**Issue by ExpandingMan** [JuliaCloud/AWS.jl#479]
I don't see any...

**Comment by omus** [comment]
Fair point. I'll try to get to this week but it may be the next week. I'll say you are free to access the fields and properties of `AWS.Response` and should only avoid using the underscore prefix functions.

**Comment by ExpandingMan** [comment]
Is this gearing up for a breaking change by any chance?  The `Response` type itself was really well done (i.e. provides essentially all of the features of whatever would have been returned previously) yet using it requires special arguments to `@service` and calling any of the services directly requires passing a `FeatureSet` to avoid a deprecation warning.  In my opinion it would be much better to simply always return the `Response` type.

If we can figure out a way to cleanup the arguments (use keyword arguments) as has been discussed elsewhere, it might be a good opportunity to put everything together for a single breaking change.
